javascript/README.md

```diff
@@ -1,4 +1,4 @@
-# TypeScript
+# JavaScript
```

Comment:
La partie JS est générée à partir de la version typescript via le script npm `to-js`: https://github.com/gladiaio/gladia-samples/blob/main/typescript/package.json 
Peut-être ajouter un sed sur la commande pour faire le changement ?

Reply:
Je vais voir pour rajouter du formatting aussi à la suite.